frontend/src/expert/expertGlobals.js

```diff
@@ -186,8 +227,17 @@ export const MIST_builtin_values = {
         "name": "m.y",
         "about": "takes the y-value of the position of the mouse on the image"
     },
+    "rad": {
```

Comment:
These don't actually seem to draw anything. Are they defined elsewhere? Used elsewhere?